guides/doc-Quickstart_Guide/master.adoc

```diff
@@ -0,0 +1,74 @@
+include::common/attributes.adoc[]
+
+:imagesdir: common/images
+:numbered:
+ifdef::foreman-el,foreman-deb,katello[]
+:toc:
+:toc-placement: left
+endif::[]
+
+:quickstart-installation-guide:
+:context: {project-context}
+:mode: connected
+:ProductName: {ProjectServer}
+
+= {project-installation-guide-title}
+
+The Foreman installer is a collection of Puppet modules that installs everything required for a full working Foreman setup. It uses native OS packaging (e.g. RPM and .deb packages) and adds necessary configuration for the complete installation.
+
+Components include the Foreman web UI, Smart Proxy, a Puppet server, and optionally Passenger, TFTP, DNS and DHCP servers. It is configurable and the Puppet modules can be read or run in “no-op” mode to see what changes it will make.
+
+include::common/modules/ref_supported-operating-systems.adoc[]
+
+The installation will require 4GB of memory, see [System Requirements] for more information.
```

Comment:
@melcorr do you know how I can add a link here to point at the Foreman or Katello system requirements in the main published installation document ?

Reply:
I genuinely forget a good way do to this. There's been a lot of change. I'll ask my former colleague @spetrosi in the morning. {InstallingProjectDocURL}l#system-requirements_foreman hard links it to Foreman, but that is not a good way.

Reply:
The following sould work both upstream and downstream:
`{InstallingProjectDocURL}system-requirements_{project-context} `